lib/device/drivers/detection.js

```diff
@@ -48,20 +48,26 @@ module.exports = {
       if (_.has(d, 'facialRecognition')) {
         o.demographics = _.reduce(d.facialRecognition, function (r, v, k) {
           // translate from { tag: 'EMOTION', emotion: 'HAPPY' to { emotion: 'happy' }
-          debug('v:', v);
+          debug("v:", v);
```

Comment:
Be consistent with quotes, if the rest of the code uses single quotes, follow consistency. (It's also JS best practice).